README.md

```diff
@@ -53,6 +53,8 @@ See [contributing instructions](CONTRIBUTING.md) to help improve this project.
   * [`manual-workspace-info` command](#manual-workspace-info-command)
   * [`create-account-groups` command](#create-account-groups-command)
   * [`validate-groups-membership` command](#validate-groups-membership-command)
+  * [`cluster-remap` command](#cluster-remap)
+  * [`revert-cluster-remap` command](#revert-cluster-remap)
```

Comment:
this anchor is invalid and non-clickable. regenerate with markdown IDE plugin:
<img width="750" alt="image" src="https://github.com/databrickslabs/ucx/assets/259697/5f05d4b5-23cb-4b2e-8332-d97fe5310124">

Reply:
Updated the command